hybrisCustom/custom-ext/bonstore/bonstorecore/src/org/bonstore/core/populators/EmailMessageModelPopulator.java

```diff
@@ -0,0 +1,99 @@
+/**
+ *
+ */
+package org.bonstore.core.populators;
+
+import static de.hybris.platform.servicelayer.util.ServicesUtil.validateParameterNotNull;
+
+import de.hybris.platform.acceleratorservices.model.email.EmailAddressModel;
+import de.hybris.platform.acceleratorservices.model.email.EmailMessageModel;
+import de.hybris.platform.converters.Populator;
+import de.hybris.platform.core.model.user.CustomerModel;
+import de.hybris.platform.servicelayer.dto.converter.ConversionException;
+import de.hybris.platform.servicelayer.model.ModelService;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.bonstore.core.model.OrganizationModel;
+
+
+/**
+ * @author Tanmoy_Mondal
+ *
+ */
+public class EmailMessageModelPopulator implements Populator<OrganizationModel, EmailMessageModel>
```

Comment:
Usually populators in the hybris are used for populating model->data, so this way is valid but little bit confusing